libsplinter/src/admin/client/event/ws/actix_web_client.rs

```diff
@@ -159,6 +159,7 @@ impl RunnableAwcAdminServiceEventClient {
         let last_event_id = Arc::new(AtomicU64::new(last_event_id.unwrap_or(0)));
         let received_id = last_event_id.clone();
         let received_sender = event_sender.clone();
+
```

Comment:
As this is formatting related it should be part of a separate commit (or the change should be removed)